app/assets/stylesheets/global.sass

```diff
@@ -151,8 +151,11 @@ html, body
         height: 20px
 
     &.conferences
+      font-size: 0.9em
       td
```

Comment:
@alicetragedy Okay? I like it when the table font is a bit smaller than the regular text.